modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/viewmodel/BookmarksViewModel.kt

```diff
@@ -72,19 +82,20 @@ class BookmarksViewModel
                             selectedListFlow,
                         ) { bookmarks, isMultiSelecting, selectedList ->
                             _uiState.value = if (bookmarks.isEmpty()) {
-                                UiState.Empty
+                                UiState.Empty(sourceView)
                             } else {
                                 UiState.Loaded(
                                     bookmarks = bookmarks,
                                     isMultiSelecting = isMultiSelecting,
                                     isSelected = { selectedList.contains(it) },
                                     onRowClick = ::onRowClick,
+                                    sourceView = sourceView,
                                 )
                             }
                         }.stateIn(viewModelScope)
                     } ?: run { // This shouldn't happen in the ideal world
                         Timber.e("Episode not found.")
```

Comment:
Would it be worth making this a `LogBuffer.e` call so it shows up in support logs if it happens? I don't feel strongly, just wondering.

Reply:
Changed in [e16a4cd](https://github.com/Automattic/pocket-casts-android/pull/1231/commits/e16a4cde0e261a27ddb38c69ed5fdae59d4f2773)